src/sankey/index.js

```diff
@@ -0,0 +1,137 @@
+    height: PropTypes.number.isRequired,
+    margin: PropTypes.number,
+    nodeWidth: PropTypes.number,
+    nodePadding: PropTypes.number,
+    layout: PropTypes.number,
+    align: PropTypes.oneOf(['justify', 'left', 'right', 'center']),
+    nodes: PropTypes.arrayOf(PropTypes.object).isRequired,
+    links: PropTypes.arrayOf(PropTypes.shape({
+      source: PropTypes.number.isRequired,
+      target: PropTypes.number.isRequired
+    })).isRequired,
+    hasVoronoi: PropTypes.bool,
+    onClick: PropTypes.func,
+    onHover: PropTypes.func,
+    onBlur: PropTypes.func
+  }
+
+  render() {
+
+    const {
+      width,
+      height,
+      margin,
+      nodeWidth,
+      nodePadding,
+      align,
+      layout,
+      nodes,
+      links,
+      hasVoronoi,
+      onClick,
+      onHover,
+      onBlur
+    } = this.props;
+
+    const sankeyInstance = sankey()
+      .size([width, height])
+      .nodeWidth(nodeWidth)
+      .nodePadding(nodePadding)
+      .nodes(nodes)
+      .links(links)
+      .align(align)
+      .layout(layout);
+
+    const nWidth = sankeyInstance.nodeWidth();
+    const path = sankeyInstance.link();
+
+    const voronoiInstance = voronoi()
+      .x(d => d.x + d.dx / 2)
+      .y(d => d.y + d.dy / 2)
+      .extent([[-margin, -margin], [width + margin, height + margin]]);
+
+    return (
+      <svg height={height + margin} width={width + margin} className="rv-sankey">
+        <g transform={`translate(${margin / 2}, ${margin / 2})`}>
+
+          {links.map((link, i) => (
+            <path
+              d={path(link)}
+              className="rv-sankey__link"
+              opacity={link.opacity || defaultLinkOpacity}
+              stroke={link.color || defaultLinkColor}
+              strokeWidth={Math.max(1, link.dy)}
+              fill="none"
+              id={link.id}
+              key={link.id || link.key || `link-${i}`} />
+          ))}
+
+          {nodes.map((node, i) => (
+            <g
+              transform={`translate(${node.x}, ${node.y})`}
+              className="rv-sankey__node"
+              opacity={node.opacity || defaultNodeOpacity}
+              id={node.id}
+              key={node.id || node.key || `node-${i}`}>
+              <rect
+                onClick={() => onClick(node)}
+                onMouseOver={() => onHover(node)}
+                onMouseOut={() => onBlur(node)}
+                fill={node.color || defaultNodeColor}
+                height={node.dy}
+                width={nWidth} />
+            </g>
+          ))}
+
+          {hasVoronoi && (
+            <g className="rv-sankey__voronoi">
+              {voronoiInstance.polygons(nodes).map((d, i) => (
+                <path
+                  d={`M${d.join('L')}Z`}
+                  onClick={() => onClick(d.data)}
+                  onMouseOver={() => onHover(d.data)}
+                  onMouseOut={() => onBlur(d.data)}
+                  fill="none"
+                  style={{pointerEvents: 'all'}}
```

Comment:
why this? also if you want to use style, please use the scss it make it easier for users to override things for themselves